x/concentrated-liquidity/clmocks/listeners.go

```diff
@@ -0,0 +1,32 @@
+package clmocks
+
```

Comment:
Was this manually generated? Just curious

Reply:
Yes, manual add